examples/node/pop_demo.ts

```
@@ -426,10 +489,424 @@ function rectangle_roi_area(arg: string): Area {
     }
 }
 
+function tuple_of_numbers(arg: string, expected: number, option: string): number[] {
+  const parsed = eval(arg.replace('(', '[').replace(')', ']'))
```

Biome (2.5.8)

[error] eval() exposes to security risks and performance issues.

(lint/security/noGlobalEval)